kairos-cli/bin/main.rs

```diff
@@ -1,3 +1,46 @@
+mod commands;
+mod common;
+mod crypto;
+mod error;
+
+use clap::Command;
+use commands::{Deposit, Transfer, Withdraw};
+use std::process;
+
+fn cli() -> Command {
+    Command::new("Kairos Client")
+        .about("CLI for interacting with Kairos")
+        .subcommand(Deposit::new_cmd())
+        .subcommand(Transfer::new_cmd())
+        .subcommand(Withdraw::new_cmd())
```

Comment:
Imagine someone adding a new subcommand, not being aware that this needs to be extended.

Reply:
This got fixed with refactor: https://github.com/cspr-rad/kairos/pull/17/commits/f54366613e48e8b5f9f5410a26b2b43124965881.